applications/RANSApplication/custom_utilities/rans_variable_utilities.cpp

```diff
@@ -28,13 +27,14 @@ namespace Kratos
 {
 namespace RansVariableUtilities
 {
-void ClipScalarVariable(unsigned int& rNumberOfNodesBelowMinimum,
-                        unsigned int& rNumberOfNodesAboveMaximum,
-                        unsigned int& rNumberOfSelectedNodes,
-                        const double MinimumValue,
-                        const double MaximumValue,
-                        const Variable<double>& rVariable,
-                        ModelPart& rModelPart)
+void ClipScalarVariable(
+    unsigned int& rNumberOfNodesBelowMinimum,
+    unsigned int& rNumberOfNodesAboveMaximum,
+    unsigned int& rNumberOfSelectedNodes,
```

Comment:
Just as a comment for a future minor improvement.

I'd return a tuple containing the number of nodes below and above the maximum (I assume the number of selected nodes is the summation of these two magnitudes) to avoid passing these references.

Reply:
i will do this in Step 3.3 :) This is used there.

Reply:
done in Step 3.3 :)